.github/workflows/build_packages.yaml

```diff
@@ -94,6 +94,11 @@ jobs:
         }
         make ensure-rebar3
         make ${{ matrix.profile }}
+        ## Delete outdated cert store
+        $Cert = "_build/${{ matrix.profile }}/rel/emqx/lib/certifi-2.9.0/priv/cacerts.pem"
```

Comment:
here and below, is it possible to make certifi version a wildcard, or otherwise detect it from dependencies?
So that we won't have regression when we bump certifi version.

Reply:
Updated with wildcard for version.